lib/VeresOneClient.js

```diff
@@ -106,25 +106,32 @@ class VeresOneClient {
   }
 
   async getTicketServiceProof({operation, ticketService}) {
-    const baseURL = ticketService;
-    const ticketServiceApi = create({baseURL});
-    const response = await ticketServiceApi.post(
-      '/', {operation}, {httpsAgent: this.httpsAgent});
-    if(response.problem) {
+    let result;
+    try {
+      // note: ky will add / to the end of the url
+      result = await httpClient.post('', {
+        prefixUrl: ticketService,
+        json: {operation}, httpsAgent: this.httpsAgent
```

Comment:
Shouldn't the param be `agent` instead of `httpsAgent` here? Also, can we move the httpsAgent to the next line.

Reply:
Also, I've written elsewhere that I would like to not sue the `prefixUrl` API.

Reply:
removed `httpsAgent` here: https://github.com/veres-one/did-veres-one/pull/65/commits/69e371ec0a14ac645f6c498a012114b4edf19f26